Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -974,6 +980,36 @@ public ItSystemUsageValidationResult CheckSystemValidity()
             return new ItSystemUsageValidationResult(errors);
         }
 
+        public Maybe<OperationError> AddPersonalData(GDPRPersonalDataOption option)
+        {
+            if (SensitiveDataLevels.Any(x => x.SensitivityDataLevel == SensitiveDataLevel.PERSONALDATA) == false)
+            {
+                return new OperationError("You cannot add a new PersonalData option before adding SensitiveDataLevel.PersonalData", OperationFailure.BadState);
+            }
+            
+            if(PersonalDataOptions.Any(x => x.PersonalData == option))
+                return Maybe<OperationError>.None;
+
+            var personalDataOption = new ItSystemUsagePersonalData() {ItSystemUsage = this, PersonalData = option};
+            PersonalDataOptions.Add(personalDataOption);
+            return Maybe<OperationError>.None;
+        }
+
+        public Maybe<OperationError> RemovePersonalData(GDPRPersonalDataOption option)
+        {
+            var personalDataOption = PersonalDataOptions.FirstOrDefault(x => x.PersonalData == option);
```

Comment:
it is not valid to work on personal data if that sensitivity level has not been chose right?